array-manipulation-2/src/drop-right.js

```diff
@@ -0,0 +1,14 @@
+/* exported dropRight */
+// set condition for if count larger than array, return the array;
+// run a for loop (var i = 0; i <= (array.length - 1) - count; i++);
+// and push result to result array;
+
+function dropRight(array, count) {
+  if (count > array.length) {
+    return array;
+  } var result = [];
+  for (var i = 0; i <= (array.length - 1) - count; i++) {
```

Comment:
It would be more idiomatic in the test expression to use `<` instead of `<=`. To make that work, just remove the `- 1`.